cmd/migrate/v2/migrate.go

```diff
@@ -0,0 +1,129 @@
+package v2
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	v1 "github.com/forbole/juno/v3/cmd/migrate/v1"
+	"github.com/forbole/juno/v3/cmd/parse"
+
+	"gopkg.in/yaml.v3"
+
+	loggingconfig "github.com/forbole/juno/v3/logging/config"
+	"github.com/forbole/juno/v3/modules/pruning"
+	"github.com/forbole/juno/v3/modules/telemetry"
+	nodeconfig "github.com/forbole/juno/v3/node/config"
+	"github.com/forbole/juno/v3/node/remote"
+	"github.com/forbole/juno/v3/types/config"
+)
+
+// RunMigration runs the migration that migrates the data from v1 to v2
+func RunMigration(_ *parse.Config) error {
+	v2Config, err := migrateConfig()
+	if err != nil {
+		return nil
```

Comment:
@RiccardoM  Is this made on purpose to return `nil` while `err != nil` ?

Reply:
nope, my bad. Fixed it